sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQuerySuite.scala

```diff
@@ -958,11 +959,11 @@ class StreamingQuerySuite extends StreamTest with BeforeAndAfter with Logging wi
     // Ideally we should copy "_spark_metadata" directly like what the user is supposed to do to
     // migrate to new version. However, in our test, "tempDir" will be different in each run and
     // we need to fix the absolute path in the metadata to match "tempDir".
-    val sparkMetadata = FileUtils.readFileToString(new File(legacySparkMetadataDir, "0"), "UTF-8")
+    val sparkMetadata = FileUtils.readFileToString(new File(legacySparkMetadataDir, "0"),
+      StandardCharsets.UTF_8)
```

Comment:
Shall we import `java.nio.charset.StandardCharsets.UTF_8` instead of splitting this into two lines?